sentry-ruby/lib/sentry/rack/capture_exceptions.rb

```diff
@@ -53,8 +55,10 @@ def transaction_op
         "rack.request".freeze
       end
 
-      def capture_exception(exception, _env)
-        Sentry.capture_exception(exception)
+      def capture_exception(exception, env)
+        Sentry.capture_exception(exception).tap do |event|
```

Comment:
@grosser This also has the same logic.
I had to replicate them in 2 places because the receivers are different: `Sentry` and `Sentry::Rails` and I don't want to introduce another abstraction just for the 2 occurrences.